### PR TITLE
Lollo in

### PR DESCRIPTION
**Pull request by lorenzoCapelli**

### CODE REVIEW

File: peepholelib/models/model_wrap.py [hunk 1]
```diff
@@ -5,6 +5,40 @@
 # torch stuff
 import torch
 
+import torch.nn as nn
+from collections import OrderedDict
+from torch import Tensor
+
+means = {
```
Comment (lsrosa) [inline peepholelib/models/model_wrap.py]: move these values to the transforms.py

File: peepholelib/models/model_wrap.py [hunk 2]
```diff
@@ -208,6 +242,25 @@ def load_checkpoint(self, **kwargs):
...
+        Args:
+        - mean (torch.tensor): mean for each channel
+        - std (torch.tensor): std for each channel
+        '''
+
+        dss = kwargs['dataset']
+
+        mean = means[dss].to(self.device)
+        std = stds[dss].to(self.device)
+        layers = OrderedDict([('normalizer', ImageNormalizer(mean, std)),
```
Comment (lsrosa) [inline peepholelib/models/model_wrap.py]: fix line breaks and indentations

File: peepholelib/models/model_wrap.py [hunk 3]
```diff
@@ -5,6 +5,40 @@
...
+        }
+
+'''Inspired by https://github.com/RobustBench/robustbench/blob/master/robustbench/model_zoo/architectures/utils_architectures.py'''
+
+class ImageNormalizer(nn.Module):
+
+    def __init__(self, mean, std):
+        super(ImageNormalizer, self).__init__()
+
+        self.register_buffer('mean', mean.view(1, 3, 1, 1))
```
Comment (lsrosa) [inline peepholelib/models/model_wrap.py]: These `view()` calls should be unnecessary; the means and stds should be passed in the correct format. This will make it easier to generalize this module to an "input normalizer"

**Review by lsrosa — CHANGES_REQUESTED** [review]
Please start writing more descriptive names for the PR and the commits.

File: peepholelib/models/model_wrap.py [hunk 4]
```diff
@@ -5,6 +5,40 @@
...
+        self.register_buffer('std', std.view(1, 3, 1, 1))
+
+    def forward(self, input: Tensor) -> Tensor:
+        return (input - self.mean) / self.std
+
+    def __repr__(self):
+        return f'ImageNormalizer(mean={self.mean.squeeze()}, std={self.std.squeeze()})'
```
Comment (lsrosa) [inline peepholelib/models/model_wrap.py]: fix line break and indentation

**Review by lsrosa — CHANGES_REQUESTED** [review]

File: peepholelib/models/model_wrap.py (hunk 4 above)
Comment (lsrosa) [reply peepholelib/models/model_wrap.py]: sorry, it was a bad folding from the browser viewer. Ignore

File: peepholelib/models/model_wrap.py [hunk 5]
```diff
@@ -208,6 +242,25 @@ def load_checkpoint(self, **kwargs):
...
+    def normalize_model(self, **kwargs):
+        '''
+        Wrap the model with an ImageNormalizer layer at the beginning.
+        Args:
+        - mean (torch.tensor): mean for each channel
+        - std (torch.tensor): std for each channel
+        '''
+
+        dss = kwargs['dataset']
+
```
Comment (lsrosa) [inline peepholelib/models/model_wrap.py]: add arguments for passing the `mean` and `std` here, and remove the `dss` argument. So the user can import the values from `transforms.py` and then to this method.
